torch/onnx/_internal/fx/passes/functionalization.py

```diff
@@ -102,15 +104,29 @@ def _run(self, *args) -> torch.fx.GraphModule:
         module = _utils.wrap_graph_module_for_node_meta_preservation(self.module)
 
         functionalized_callable = self._functionalize(module)
-        fx_mode = "symbolic" if self.enable_dynamic_axes else "fake"
-
-        graph_module = proxy_tensor.make_fx(
-            functionalized_callable,
-            decomposition_table={},
-            tracing_mode=fx_mode,
-            _allow_non_fake_inputs=True,
-            _allow_fake_constant=self.allow_fake_constant,
-        )(*args)
+
+        # Mimic `torch._dynamo.export(aten_graph=True)` behavior in invoking `make_fx`.
+        # TODO: May need revisit for user fake mode export + dynamic shape scenario.
+        fake_mode: Optional[fake_tensor.FakeTensorMode] = self.fake_mode
+        maybe_fake_args = self._maybe_fakefy_args(fake_mode, *args)
+        if fake_mode is not None:
+            # Using existing fake mode as context, signal `make_fx` that it does not need
+            # to create a new fake mode by passing tracing_mode as "real".
+            tracing_mode = "real"
+        else:
+            # Existing fake mode not found, signal `make_fx` to create one.
+            fake_mode = contextlib.nullcontext()  # type: ignore[assignment]
+            tracing_mode = "symbolic" if self.enable_dynamic_axes else "fake"
+
+        assert fake_mode is not None
```

Comment:
```suggestion
        assert fake_mode is not None  # for mypy
```

Reply:
Used github commit changes for this. I'm not sure if it gets picked up by ghstack and merge bot, let's find out.

Reply:
ok that didn't work